web/styles/inbox.css

```diff
@@ -528,3 +523,43 @@
     text-decoration: underline;
     color: hsl(200deg 100% 40%);
 }
+
+#inbox-filter_widget {
+    margin-left: 5px;
+    height: 30px;
+    max-width: var(--width-inbox-filters-dropdown);
+    border: 1px solid var(--color-border-inbox-search);
+    background-color: var(--color-background-inbox-search);
+    gap: 5px;
+
+    &:hover {
+        background-color: var(--color-background-inbox-search-hover);
+        border: 1px solid var(--color-border-inbox-search-hover);
+    }
+
+    &:focus {
+        outline: none;
+    }
+
+    &:focus-visible {
+        outline: 2px solid var(--color-outline-focus);
+    }
+}
+
+#inbox-filter_widget .dropdown_widget_value {
+    text-overflow: ellipsis;
+    white-space: nowrap;
+    overflow: hidden;
+    width: 100%;
+    text-align: left;
+}
+
+#inbox-filter_widget .fa-chevron-down {
+    color: var(--color-icons-inbox);
+    opacity: 0.4;
+}
```

Comment:
It's usually better to just use dashes consistently in a name rather than mixing dashes and underscores.

Reply:
Opened #27658